python/xoscar/backends/pool.py

```diff
@@ -554,7 +554,8 @@ async def on_new_channel(self, channel: Channel):
         while not self._stopped.is_set():
             try:
                 message = await channel.recv()
-            except EOFError:
+            except Exception as e:
```

Comment:
Better to specify which errors clearly.

Reply:
Should close the connection on all errors,  otherwise socket will not be gc, making a fd leak